_includes/consent-management/trustarc.html

```diff
@@ -0,0 +1 @@
+<script async="async" src="//consent.trustarc.com/notice?domain=docker.com&c=teconsent&js=nj&noticeType=bb&gtm=1"></script>
```

Comment:
Looks like this can only work when hosted on a `docker.com` domain; the script does;

```
(new Image(1,1)).src=("https://consent.trustarc.com/log".replace("http:","https:"))+"?domain=docker.com&country=nl&state=&behavior=implied&c="+
```

Reply:
The new testing site looks like it would need to be HTTPS for this to work, right?

Reply:
Answering a question above - people in EU have a different compliance standard. Everybody gets to opt out, EU is a bit more stringent - and California has their own compliance requirements as well. The IP address of the user determines which flow they experience.

Reply:
@thaJeztah - do we have a way to set up the staging / test area as HTTPS so we can get the script to load?

Reply:
(for those reading along) done ☝️ (I sent a link through slack)